acme/setup.py

```diff
@@ -76,6 +83,7 @@
     include_package_data=True,
     install_requires=install_requires,
     extras_require={
+        ':python_version<"2.7.9"': py279_extras,
```

Comment:
For future reference, the trouble turned out to be that setuptools doesn't yet support the comparators < and > in env markers.